res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -804,6 +804,12 @@
    macOS packaging: Enable app sandbox in ad-hoc-packaged (i.e. non-notarized) bundles too
    #12101
   </li>
+  <li>
+   Drop support for macOS versions earlier than 11, Windows 7, Windows 10 versions up to 1803 and Ubuntu 20.04 "Focal Fossa"
```

Comment:
```suggestion
   Drop support for macOS versions earlier than 11
   Drop support for Windows versions earlier than Windows 10 build 1809
   Drop support for Ubuntu versions earlier than 22.04
```
Let's write this in consistent style, not "earlier than" and "up to" mixed in one sentence.